model/riscv_sys_control.sail

```diff
@@ -408,8 +482,187 @@ function handle_exception(e: ExceptionType) -> unit = {
   set_next_pc(exception_handler(cur_privilege, CTL_TRAP(t), PC))
 }
 
-function handle_interrupt(i : InterruptType, del_priv : Privilege) -> unit =
+function handle_interrupt(i : InterruptType, del_priv : Privilege) -> unit = {
+  IEtrigger_match(cur_privilege, num_of_interruptType(i), false);
   set_next_pc(trap_handler(del_priv, true, interruptType_to_bits(i), PC, None(), None()))
+}
+
+function find_first_zero(v: xlenbits) -> {'n, (1 <= 'n < xlen). int('n)} = {
+  foreach (i from 1 to (xlen - 1)) {
```

Comment:
Why from 1 and not from 0? Are array indices 1-based?

Reply:
Fixed